pkg/tools/fs/utils_test.go

```diff
@@ -26,55 +27,89 @@ import (
 	"testing"
 	"time"
 
+	"github.com/google/uuid"
 	"github.com/stretchr/testify/require"
+	"go.uber.org/goleak"
 
 	"github.com/networkservicemesh/sdk/pkg/tools/fs"
 )
 
+const macOSName = "darwin"
+
+func checkPathError(t *testing.T, err error) error {
+	if fErr, ok := err.(*iofs.PathError); ok {
+		if u := fErr.Unwrap(); u != nil {
+			t.Log(u) // Without unwrapping printing gives only numeric error code.
+		}
+	}
+
+	return err
+}
```

Comment:
As I can see this function just logs PathError. 
Can we just use err.Error() to get information about the error?
```suggestion
```